inc/html.php

```diff
@@ -2372,3 +2372,5 @@ function html_tab($href, $caption, $selected=false) {
     echo $tab;
 }
 
+
```

Comment:
It appears this is just a white-space change, not related to this issue. Please revert this.